docs/index.yml

```diff
@@ -0,0 +1,148 @@
+### YamlMime:Hub
+title: NuGet documentation
+summary: NuGet is the package manager for .NET. It enables developers to create, share, and consume useful .NET libraries. NuGet client tools provide the ability to produce and consume these libraries as "packages".
+brand: dotnet
+metadata:
+   title: NuGet documentation | Microsoft Docs
+   titleSuffix: ""
+   description: NuGet is the package manager for Microsoft development platforms including .NET. The NuGet client tools provide the ability to create and consume packages.
+   ms.topic: hub-page
+   author: karann-msft
+   ms.author: karann
+   ms.date: 01/13/2020
+
+highlightedContent:
+   items:
+   - title: What is NuGet?
+     itemType: overview
+     url: /nuget/what-is-nuget
+   - title: Install NuGet client tools
+     itemType: download
+     url: /nuget/install-nuget-client-tools
+   - title: Install and use a package
+     itemType: get-started
+     url: /nuget/quickstart/install-and-use-a-package-using-the-dotnet-cli
+   - title: NuGet.org
+     itemType: overview
+     url: /nuget/nuget-org/overview-nuget-org
+
+
+conceptualContent:
+  title: Consume, create, and publish packages
+  items:
+    - title: Get started
+      links:
+        - text: Install NuGet client tools
+          itemType: download
+          url: /nuget/install-nuget-client-tools
+        - text: Install and use a package - dotnet CLI
+          itemType: quickstart
+          url: /nuget/quickstart/install-and-use-a-package-using-the-dotnet-cli
+        - text: Install and use a package - Visual Studio
+          itemType: quickstart
+          url: /nuget/quickstart/install-and-use-a-package-in-visual-studio
+        - text: Create a package - dotnet CLI
```

Comment:
These next three `Create a package - ...` links are repeated with the `Create packages` section:

![image](https://user-images.githubusercontent.com/737941/72848769-2a24cf00-3c5a-11ea-8718-7d354734329e.png)

Is that intentional? Should we deduplicate these links?

Reply:
@loic-sharma The "Get started" section are intended to be links to the Quickstarts. The "Consume" and "Create" sections link to the main reference/how to articles. The "Use Visual Studio" link is supposed to go to the quickstart, only because there's not an equivalent reference/how to (I'll fix the link). I'm not sure this is a gap in this case as much as avoiding redundant content. On this page, it has the appearance of a gap...

Reply:
Ok sounds reasonable, thanks for the info!